ext/datastore/bigquery/table.go

```diff
@@ -13,12 +18,22 @@ import (
 
 var (
 	tableNameParseRegex = regexp.MustCompile(`^([\w-]+)\.(\w+)\.([\w-]+)$`)
+	errorReadTableSpec  = "failed to read table spec for bigquery"
+)
+
+const (
+	BackupConfigDataset  = "dataset"
+	BackupConfigPrefix   = "prefix"
+	BackupConfigTTL      = "ttl"
```

Comment:
This `ttl` is very ambiguous without documentation, that is, this is in hours, minutes, dates, seconds, etc? Should we think of a better name here? Also, day size changes based on where we are, should we take duration instead? So it will be a proper parsable `go` `time.Duration` type? That is users can provide values like `720h` for 30 days or `24h` or even `60m`. What do you think?

Reply:
totally agree. updated.